packages/paste-design-tokens/tokens/global/box-shadow.yml

```diff
@@ -87,8 +87,11 @@ props:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-0-transparent-50}"
     comment: Shadow border on interactive elements on inverse backgrounds.
   shadow-border-inverse-strong:
-    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-0-transparent-90}"
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-0-transparent-70}"
     comment: Strong shadow border on interactive elements on inverse backgrounds.
+  shadow-border-inverse-stronger:
+    value: "{palette-gray-0-transparent-90}"
```

Comment:
```suggestion
    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-0-transparent-90}"
```

Reply:
This one is just missing the offsets, since it's a shadow token (which was my bad, I wrote it wrong in the AC).